### PR TITLE
Skeleton for Fate backend

### PR DESCRIPTION
**Pull request by happi**

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -1,8 +1,11 @@
 {erl_opts, [debug_info]}.
 
 {deps, [ {aebytecode, {git, "https://github.com/aeternity/aebytecode.git",
-                            {branch,"fortuna"}}}
+                             {branch,"fortuna"}}}
        , {getopt, "1.0.1"}
+        % waiting for https://github.com/jlouis/enacl/pull/40 to be merged
```
Comment (hanssv) [inline rebar.config]: This should not be here?

Comment (happi) [reply rebar.config]: Removed enacl.

File: rebar.config [hunk 2]
```diff
@@ -1,14 +1,14 @@
 {erl_opts, [debug_info]}.
 
 {deps, [ {aebytecode, {git, "https://github.com/aeternity/aebytecode.git",
-                            {branch,"fortuna"}}}
+                             {tag,"v2.0.1"}}}
```
Comment (hanssv) [inline rebar.config]: This reminds me again, we need to think about versioning (and documentation, and release notes?!)...

File: rebar.config [hunk 3]
```diff
@@ -25,3 +25,11 @@
             {plt_apps, all_deps},
             {base_plt_apps, [erts, kernel, stdlib, crypto, mnesia]}
            ]}.
+
+{relx, [{release, {aesophia, "2.0.0"},
```
Comment (hanssv) [inline rebar.config]: What are we releasing?

**Pull request merged**